src/Atoms/ContestDetailsLine/ContestDetailsLine.js

```diff
@@ -0,0 +1,21 @@
+import ContestDetailsLineStyled from './ContestDetailsLineStyled';
+import PropTypes from 'prop-types';
+
+const ContestDetailsLine = ({ text, highlight = false, judge = false }) => {
+  let textArray = [];
+  typeof text == 'string' ? textArray.push(text) : (textArray = text);
+  return (
+    <ContestDetailsLineStyled highlight={highlight} judge={judge}>
+      {textArray.map((text, index) => (
+        <div key={index}>{text}</div>
+      ))}
+    </ContestDetailsLineStyled>
+  );
+};
+
+ContestDetailsLine.propTypes = {
+  text: PropTypes.oneOfType([PropTypes.string, PropTypes.array]),
+  highlight: PropTypes.bool,
+  judge: PropTypes.bool,
```

Comment:
Similar to https://github.com/CC2021-WBL/GOOD-BOI-Application/pull/202/files#r798045688.